src/App.js

```diff
@@ -47,27 +43,30 @@ function App() {
   console.log("Player Index: " + playerIndex);
   console.log(playerNames);
   return (
+    
     <Router>
       <div>
         {/* A <Switch> looks through its children <Route>s and
           renders the first one that matches the current URL. */}
-        <Switch>
-          <Route exact path="/">
-            <div align="center" style={{ position: "absolute", top: "0", bottom: "0", left: "0", right: "0", margin: "auto" }}>
-              <LoginComponent setPlayerID={setPlayerID} setHost={setHost} setRoomName={setRoomName} />
-            </div>
-          </Route>
-          <Route exact path="/start">
-            <div align="center">
-              <MainGameScreen playerID={playerID} isHost={isHost} roomName={roomName} playerIndex={playerIndex} playerNames = {playerNames}/>
-            </div>
-          </Route>
-          <Route exact path="/GameStart">
-            <div align="center">
-              <GameStart playerID={playerID} isHost={isHost} roomName={roomName} setPlayerIndex={setPlayerIndex} setPlayerNames = {setPlayerNames}/>
-            </div>
-          </Route>
-        </Switch>
+        <RoomContextProvider>
```

Comment:
So if we have multiple contexts, how would it look? Like would it be nested conext tags? 
Like: 
<playerIdContext>
      <RoomContext> etc

Reply:
ya you could nest

Reply:
player context is also only needed in a few places so we could just wrap those